corehq/apps/api/resources/v0_5.py

```diff
@@ -1089,26 +1095,296 @@ def prepend_urls(self):
 
 
 class MessagingEventResource(HqBaseResource, ModelResource):
```

Comment:
Could this get a docstring clarifying that it returns messaging subevents?

Reply:
1251af68a6c